src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -2002,7 +2002,11 @@ bool isRefEscape
 
                     if (!valid)
                     {
-                        ReportInvocationEscapeError(syntax, symbol, parameter, checkingReceiver, diagnostics);
+                        if (symbol is not SignatureOnlyMethodSymbol)
```

Comment:
I didn't understand why we skip reporting an error in this case. Could you please explain?

Reply:
> I didn't understand why we skip reporting an error in this case. Could you please explain?

This error is for the entire invocation, and, it looks to me, that in the scenarios of interest this will be a second error. Reported in addition to an error about specific argument exposure. Since the methods are synthesized, the error about them won't be meaningful to a user. It will be just a nose, or a source of confusion.